docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.0'
+
+services:
+  tests:
+    build: .
+    container_name: php-imap
+    volumes:
+      - .:/project_orig
+    command: bash -c "wait-for-it.sh imap.server:993 && cp -dpr /project_orig /project && cd /project && composer install && vendor/bin/phpunit"
```

Comment:
@LeadTechVisas I've replaced `sleep` with [wait-for-it](https://github.com/vishnubob/wait-for-it), maybe it's helpful for you to know this